crates/circuit/src/lib.rs

```diff
@@ -37,53 +37,60 @@ use pyo3::prelude::*;
 use pyo3::types::{PySequence, PyTuple};
 use pyo3::PyTypeInfo;
 
-pub type BitType = u32;
```

Comment:
Is the reason to remove this because it's used for more than bits now?

Reply:
Pretty much, yeah.

I'd introduced it originally, but it doesn't feel very Rust-y. I think it wasn't doing much for us.